ksqldb-api-client/src/main/java/io/confluent/ksql/api/client/impl/QueryResponseHandler.java

```diff
@@ -15,8 +15,7 @@
 
 package io.confluent.ksql.api.client.impl;
 
-import com.fasterxml.jackson.databind.ObjectMapper;
-import io.confluent.ksql.api.client.util.JsonMapper;
+import io.confluent.ksql.rest.ApiJsonMapper;
```

Comment:
I'm guessing that the answer is "yes", but do we still need the Jackson databind dependency?

Reply:
I believe we do still need it - but I'm not 100% sure. I know that some parts of the code rely on the databind mapper (the `vertx.Buffer.toJson` for example defaults to Databind if the library is present)

https://github.com/eclipse-vertx/vert.x/blob/b284c07b4578b5cd5a6e39b153422c0148459226/src/main/java/io/vertx/core/json/jackson/JacksonFactory.java#L25-L34

I was of two minds of whether to just implement our own `JsonFactory` based on our `ApiJsonMapper` and plug into their spi but decided against it (I had actually implemented it) because vertx is a bit inconsistent about using it unfortunately  (see: https://github.com/eclipse-vertx/vert.x/blob/b284c07b4578b5cd5a6e39b153422c0148459226/src/main/java/io/vertx/core/json/impl/JsonUtil.java#L37-L50 where they don't care about what you've specified, this caused me much grief in the migration haha)

Reply:
Sounds good.